boards/arm/stm32/stm32f401rc-rs485/src/stm32_bringup.c

```diff
@@ -328,5 +332,15 @@ int stm32_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_WS2812 && defined(CONFIG_WS2812_LED_COUNT)
```

Comment:
```suggestion
#if defined(CONFIG_WS2812) && defined(CONFIG_WS2812_LED_COUNT)
```

Reply:
Hi @pkarashchenko,
Thanks for your suggestion, I've pushed a new patch with your suggestion included.